daemon/src/taker_cfd.rs

```diff
@@ -363,8 +368,7 @@ impl Actor {
         &mut self,
         announcements: oracle::Announcements,
     ) -> Result<()> {
-        tracing::debug!("Updating latest oracle announcements");
```

Comment:
😭 Why don't you like my arbitrary, useless debug statements?

Reply:
hahaha, I think I actually removed that one by accident - in the maker I kept it 🤦‍♂️